bluemira/base/builder.py

```diff
@@ -88,7 +88,7 @@ class Builder(abc.ABC):
     _params: ParameterFrame
     _design_problem = None
 
-    def __init__(self, params, build_config: BuildConfig, **kwargs):
+    def __init__(self, params: Dict[str, Any], build_config: BuildConfig, **kwargs):
```

Comment:
still kwargs in here; is this right?

Reply:
just this now from my side; otherwise all good!

Reply:
Things break when I remove them. The issue is that the constructors in the concrete builder classes delegate most of the work to reinitialise. Without the kwargs, they can't pass through the parameters.

Dan and I have talked about this and he's going to open an issue to look into alternatives to `reinitialise`. 

Are we happy to leave this as is for the moment, then remove the kwargs when we address what to do with `reinitialise`? It's proving a bit of a rabbit-hole implementing a work-around.

Reply:
OK, fair does